src/pages/api/food/[category].js

```diff
@@ -0,0 +1,17 @@
+const apiKey = process.env.SPOONACULAR_API_KEY;
+const baseUrls = {
+	recipes: `https://api.spoonacular.com/recipes/complexSearch`,
+	ingredients: `https://api.spoonacular.com/food/ingredients/search`,
+};
+export default async function handler(request, response) {
+	const {number, query, max_fat, category} = request.query;
+	const baseUrl = baseUrls[category];
+	const url = `${baseUrl}?apiKey=${apiKey}&query=${query}&maxFat=${max_fat}&number=${number}`;
```

Comment:
does this work, I have some problems understanding :)

Reply:
I get my data here from the Spoonacular API. but I still have to work in it. Right now if you submit a recipe e.G. Pasta in the Food Formular 
<img width="380" alt="Bildschirmfoto 2022-10-14 um 17 28 14" src="https://user-images.githubusercontent.com/109968361/195884311-271d778a-a745-44b1-8bd9-243064ad8060.png">


then you youll get Data but just inside the console in the browser Developer tools.

You can see the result here:
<img width="345" alt="Bildschirmfoto 2022-10-14 um 17 17 42" src="https://user-images.githubusercontent.com/109968361/195883738-fb499513-c188-4996-a408-134365772ba2.png">

I can also adjust the search query to search for different values like min&max fat, min&max protein, min&max calories, and many more.
Theyre listed in the Spoonacular API:
https://spoonacular.com/food-api/docs